index.js

```diff
@@ -209,6 +213,49 @@ app.get("/popup", function (req, res) {
   res.status(200).send(popupMsg);
 });
 
+const proxyMiddleware = (target, pathRewrite) =>
+  createProxyMiddleware({
+    target,
+    changeOrigin: true,
+    pathRewrite,
+    selfHandleResponse: true,
+    onProxyRes: responseInterceptor(async (responseBuffer, proxyRes) => {
+      console.log(proxyRes.headers);
+      if (proxyRes.headers["content-type"] !== "application/json;charset=utf-8")
```

Comment:
Use `.includes(...)` as the header could not include `;charset=utf-8`

Reply:
ok